perun-rpc/src/main/java/cz/metacentrum/perun/rpc/methods/AuthzResolverMethod.java

```diff
@@ -782,5 +785,273 @@ public Void call(ApiCaller ac, Deserializer parms) throws PerunException {
 			cz.metacentrum.perun.core.api.AuthzResolver.loadAuthorizationComponents(ac.getSession());
 			return null;
 		}
+	},
+
+	/*#
+	 * Get all Vos where the given user has set one of the given roles
+	 * or the given user is a member of an authorized group with such roles.
+	 *
+	 * @param user id for which Vos are retrieved
+	 * @param List of role names for which Vos are retrieved
```

Comment:
The param is still in the wrong format. It should be like this:
```
@param roles List<String> list of role names for which Vos are retrieved
```

Reply:
Fixed for all aprameters